File: java/app/src/test/java/info/lotharschulz/github/org/verifier/AppTest.java
package info.lotharschulz.github.org.verifier;

import info.lotharschulz.github.org.verifier.api.github.Utils;
import org.testng.annotations.*;
import static org.testng.Assert.*;

public class AppTest {
    @Test public void utilsHasVerifyGitHubConnection() throws Exception {
        assertNotNull(Utils.verifyGitHubConnection("empty"), "Utils should have a verifyGitHubConnection method");
    }
}

File: app/src/test/kotlin/info/lotharschulz/github/org/verifier/AppTest.kt
package info.lotharschulz.github.org.verifier

import kotlin.test.Test
import kotlin.test.assertNotNull

class AppTest {
    @Test fun greeting() {
//        assertNotNull(classUnderTest.greeting, "app should have a greeting")
        assertNotNull("classUnderTest.greeting", "app should have a greeting")
    }
}
